runtime/integration-tests/src/utils/env.rs

```diff
@@ -935,3 +935,34 @@ pub fn pass_n(env: &mut TestEnv, n: u64) -> Result<(), ()> {
 
 	Ok(())
 }
+
+mod tests {
+	use super::*;
+
+	#[tokio::test]
+	async fn env_works() {
```

Comment:
Thank you for moving this, I was also thinking about moving it in my current branch but I'll skip.